libs/src/services/creatorNode/CreatorNodeSelection.js

```diff
@@ -91,7 +91,9 @@ class CreatorNodeSelection extends ServiceSelection {
       val: { primary, secondaries: secondaries.toString(), services: Object.keys(servicesMap).toString() }
     })
 
-    console.info('CreatorNodeSelection - final decision tree state', this.decisionTree)
+    if (log) {
```

Comment:
full support as is, but we could also police our log-level better and hopefully solve the same problem

Reply:
agreed, figured this decision tree logging may be useful/desirable at current loglevel in client. just minimal impact change